### PR TITLE
Bonnie 1273 nokogiri fix

### PR DESCRIPTION
**Pull request by c-monkey**
Fixes for bundle-audit vulnerabilities in nokogiri:
- Bumped HDS version gemfile spec so dependent libraries can use it
- Fixed HDS tests that broke on macos with changes in the ordering of bundles, resulting in a varying result for HealthDataStandards::CQM::Bundle.latest_bundle_id. The solution was to add an expiration to the desired default bundle (test/fixtures/bundles/installed.json)
- Travis config updated for rename of cql4bonnie -> master_bonnie

Pull requests into Health Data Standards require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.
 
**Submitter:**
- [X] This pull request describes why these changes were made.
- [X] This PR is into the correct branch.
- [X] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1273
- [x] Internal ticket links back to this PR
- [X] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [X] Tests are included and test edge cases
- [X] Tests have been run locally and pass
- [X] Code coverage has not gone down and all code touched or added is covered.
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here:
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here:
 
**Cypress Reviewer:**
 
Name: @okeefm 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code
 
**Bonnie Reviewer:**
 
Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: test/fixtures/bundles/installed.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
 	"active" : true,
 	"effective_date" : 1293840000,
 	"measure_period_start" : 1293840000,
+	"exported" : 1,
```
Comment (okeefm) [inline test/fixtures/bundles/installed.json]: The way Cypress/`bonnie_bundler` use `exported`, it will be a string in `YYYY-MM-DD` (e.g. `2017-07-17`) format.

**Review by hossenlopp — APPROVED** [review]

**Review by okeefm — APPROVED** [review]
